rpcserver.go

```diff
@@ -1981,6 +1990,250 @@ func (r *rpcServer) SendPaymentSync(ctx context.Context,
 	}, nil
 }
 
+// SendToRoute dispatches a bi-directional streaming RPC for sending payments
+// through the Lightning Network via predefined routes passed in. A single RPC
+// invocation creates a persistent bi-directional stream allowing clients to
+// rapidly send payments through the Lightning Network with a single
+// persistent connection.
+func (r *rpcServer) SendToRoute(sendToStream lnrpc.Lightning_SendToRouteServer) error {
```

Comment:
here we should also try to reuse common code from `SendPayment`, an approach is mentioned in the comment here: https://github.com/lightningnetwork/lnd/pull/219/files#diff-01ada5680aaa18fc79cbf78b2d55a37bR1683